pkg/services/authz/rbac/service.go

```diff
@@ -60,6 +61,10 @@ type Service struct {
 	folderCache    *cacheWrap[folderTree]
 }
 
+type RBACSettings struct {
```

Comment:
Nit. RBAC is included as part of the package name already.
```suggestion
type Settings struct {
```

Reply:
Ah good catch - updated :)